pkg/apis/configuration/validation/validation.go

```diff
@@ -7,21 +7,21 @@ import (
 	"strings"
 
 	"github.com/nginxinc/kubernetes-ingress/internal/configs"
+	v1 "github.com/nginxinc/kubernetes-ingress/pkg/apis/configuration/v1"
```

Comment:
hm. is the alias `v1` required here?
same for `validation_test.go` file as well

Reply:
`goimports`